doc/docs-revamp/dev-onboarding-quick-setup.md

```diff
@@ -0,0 +1,375 @@
+- macOS: macOS Catalina (10.15) or later
+- Windows: While Windows can be used for Haskell development, it is less common and might require additional setup such as using Windows Subsystem for Linux (WSL2) to provide a Linux-compatible environment.
+
+### Tools
+
+To start developing with Haskell and Plutus, you'll need to set up your development environment with the essential tools. This section will guide you through installing and configuring the necessary components. 
+
+#### Nix
+
+Nix is a package manager that provides reproducible and reliable software environments. It is particularly useful in Plutus development for managing dependencies and ensuring consistent development environments across different machines. 
+
+While not strictly required, Nix is highly recommended because of its effective management of complex library dependencies. 
+
+To install Nix, follow the instructions on the [official Nix website](https://nixos.org/download.html). 
+After installing, configure Nix to use IOHK’s binary caches to speed up the build process. For important details and context, see these two documents: 
+- [Contributing to Plutus > Installing and setting up Nix](https://github.com/IntersectMBO/plutus/blob/master/CONTRIBUTING.adoc),
+- [Nix setup guide](https://github.com/input-output-hk/iogx/blob/main/doc/nix-setup-guide.md): instructions for installing and configuring Nix to work with projects at IOG.
+
+#### Haskell tool stack ("Stack")
+
+Stack is a cross-platform program for developing Haskell projects. It handles dependency management, building, and testing your Haskell code. 
+
+To install the Stack: 
+1. Visit the [official Haskell Stack website](https://docs.haskellstack.org/). 
+2. Follow the installation instructions for your operating system (Linux, macOS, or Windows).
+3. Verify your installation by running `stack --version` in your terminal.
+
+#### GHC (Glasgow Haskell Compiler)
+
+- GHC is the main compiler for Haskell with many extensions and optimizations. 
+- To install GHC, run `stack setup`. Stack will automatically install the appropriate version of GHC for your project. 
+
+#### Cabal
+
+Cabal is a system for building and packaging Haskell libraries and programs. It works well with Stack and is used to manage project dependencies and build configurations. Cabal is installed automatically when you install Stack. 
+
+To verify your Cabal installation, run `cabal –version`. 
+
+### Git
+
+Make sure you have Git installed and configured. See [https://git-scm.com/](https://git-scm.com/). 
+
+### IDE
+
+An IDE provides a comprehensive environment for writing, testing, and debugging your Haskell code. Visual Studio Code (VS Code) is a widely used code editor that supports the Haskell extension. See [https://code.visualstudio.com/](https://code.visualstudio.com/). 
+
+## Testing your environment setup
```

Comment:
I don't think we need this. We are basically teaching people how to write hello world in Haskell. That's not our responsibility - we should assume users already know Haskell, and if they are new to Haskell, they can find this information (and much more materials for Haskell beginners) elsewhere.

Reply:
OK, good point.